benchmarks/graphics_pipeline/main.cpp

```diff
@@ -196,6 +191,12 @@ void ProjApp::Config(ppx::ApplicationSettings& settings)
 
 void ProjApp::InitKnobs()
 {
+    pKnobVs = GetKnobManager().CreateKnob<ppx::KnobDropdown<std::string>>("vs", 0, kAvailableVsShaders);
```

Comment:
This will break internal testing in an invisible manner. (Not your fault, due to the way we handle flags in BigWheels).

Could you do the following:
Add a check to crash if cl_options contains "vs-shader-index" and "ps-shader-index" saying it's has been removed, and no user should rely on those?
This way it will crash all internal CIs, and we won't miss this failure.